test/process_client.py

```diff
@@ -27,58 +32,160 @@ def wait_for_rest(url, max_attempts, sleeping_time, keys_to_check):
 def publish_layer(username,
                   layername,
                   file_paths=None,
-                  headers=None):
+                  headers=None,
+                  access_rights=None,
+                  title=None,
+                  ):
+    title = title or layername
     headers = headers or {}
     file_paths = file_paths or ['tmp/naturalearth/110m/cultural/ne_110m_admin_0_countries.geojson']
+
     rest_url = f"http://{settings.LAYMAN_SERVER_NAME}/rest"
 
-    r_url = f"{rest_url}/{username}/layers"
-    for fp in file_paths:
-        assert os.path.isfile(fp)
-    files = []
-    try:
-        r = requests.post(r_url,
-                          files=[('file', (os.path.basename(fp), open(fp, 'rb'))) for fp in file_paths],
-                          data={'name': layername, },
-                          headers=headers)
-        assert r.status_code == 200, r.text
-    finally:
-        for fp in files:
-            fp[0].close()
+    with app.app_context():
+        r_url = f"{rest_url}/{username}/layers"
+        for fp in file_paths:
+            assert os.path.isfile(fp)
+        files = []
+        try:
+            data = {'name': layername,
```

Comment:
Přidat files=[('file', (os.path.basename(fp), open(fp, 'rb'))) for fp in file_paths] a následně používat files.